src/services/ExperianService.ts

```diff
@@ -178,16 +221,28 @@ export class ExperianService {
     		});
     		throw new AppError(HttpCodesEnum.SERVER_ERROR, "Error retrieving Experian token");
     	}
-    	return token.Item as StoredExperianToken;
+		
+    	if (token.Item) {
+    		return token.Item as StoredExperianToken;
+    	} else { 
+    		return undefined;
+    	}
     }
 
-    checkExperianToken(token: StoredExperianToken): boolean {
+    checkExperianToken(token: StoredExperianToken | undefined): boolean {
+		
+    	if (token === undefined) {
```

Comment:
I think if the token is null/undefined we should never even be calling this function right? You don't need to check is a non-existent value is valid because it doesn't exist.

Reply:
This is a good point - will make a note to refactor